FlowCrypt/Functionality/Services/Folders Services/RemoteFoldersProviderType/Imap+folders.swift

```diff
@@ -27,18 +28,24 @@ extension Imap: RemoteFoldersProviderType {
                     guard let folders = value as? [MCOIMAPFolder] else {
                         return reject(AppErr.cast("[MCOIMAPFolder]"))
                     }
-                    resolve(folders.compactMap(FolderObject.init))
+
+                    // TODO: - Ticket - rework usage of EncryptedStorage().activeUser
+                    let folderObjects = folders.compactMap { FolderObject(with: $0, user: EncryptedStorage().activeUser) }
+                    resolve(folderObjects)
             }
         }
     }
 }
 
 // MARK: - Convenience
 private extension FolderObject {
-    convenience init?(with folder: MCOIMAPFolder, user: UserObject) {
-        guard let name = folder.name else { return nil }
+    convenience init?(with folder: MCOIMAPFolder, user: UserObject?) {
+        guard let user = user else {
+            Logger.logError("Can't initialise FolderObject without user")
+            return nil
+        }
```

Comment:
Why not make the user object required?

Reply:
Need to rework this. Filled a separate issue. (https://github.com/FlowCrypt/flowcrypt-ios/issues/438)
If you want I can fix it in scope of this pr

Reply:
Got it, can do it in separate issue - thank you. I'll mark this conversation as resolved.